test/kubernetes_test.go

```diff
@@ -479,8 +848,70 @@ func TestKubernetesIntegrationFallthrough(t *testing.T) {
 	drop 0
     }
 `
-	cases := append(dnsTestCases, dnsTestCasesFallthrough...)
-	doIntegrationTests(t, corefile, cases)
+	doIntegrationTests(t, corefile, dnsTestCasesFallthrough)
+}
+
+func getDynamicEndpoints(Qname, namespace, name string) []dns.RR {
+	rr := []dns.RR{}
+
+	result := strings.Split(getCmdOutput(name, namespace), " ")
+	len_result := len(result)
+
+	if len_result > 1 {
+		for i := 0; i < len_result; i++ {
+			rr = append(rr, test.A(Qname+"    0    IN      A   "+result[i]))
+		}
+	} else {
+		rr = append(rr, test.A(Qname+"    0    IN      A   "+result[0]))
+	}
+
+	return rr
+}
+
+func getSRVrecords(Qname, Qtype, namespace, name string) []dns.RR {
+	rr := []dns.RR{}
+
+	result := strings.Split(getCmdOutput(name, namespace), " ")
+	len_result := len(result)
+
+	if len_result > 1 {
+		for i := 0; i < len_result; i++ {
+			ip := strings.Replace(result[i], ".", "-", -1)
+			if Qtype == "TypeSRV" {
+				rr = append(rr, test.SRV(Qname+"   303    IN    SRV 0 33 1234  "+ip+"."+namespace+"."+name+".svc.cluster.local."))
+			} else {
+				rr = append(rr, test.A(ip+"."+namespace+"."+name+".svc.cluster.local.	0	IN	A	"+result[i]))
+			}
+		}
+	} else {
+		ip := strings.Replace(result[0], ".", "-", -1)
+		rr = append(rr, test.SRV(Qname+"   303    IN    SRV 0 33 1234  "+ip+"."+namespace+"."+name+".svc.cluster.local."))
+	}
+	return rr
+}
+
+func getCmdOutput(name, namespace string) string {
+	var (
+		cmd    []byte
+		cmdOut []byte
+		err    error
+	)
+	kctl := os.Getenv("$KUBECTL")
+	if cmd, err = exec.Command("env").Output(); err != nil {
+		println(os.Stderr, "There was an error running kubectl command: ", err.Error())
+	}
+	cmd_str := string(cmd)
+	println(cmd_str)
```

Comment:
I updated the title with WIP, to reflect that this is a work in progress.

Sandeep is debugging a travis vs local environment issue at the moment, so he is committing changes to the PR to debug on travis.  i.e. yes - it is debug...